repos/system_upgrade/common/actors/yumconfigscanner/libraries/yumconfigscanner.py

```diff
@@ -0,0 +1,45 @@
+from leapp.libraries.common.config.version import get_source_major_version
+from leapp.libraries.stdlib import api, run
+from leapp.models import YumConfig
+
+
+def scan_enabled_yum_plugins():
+    """
+    Runs the `yum` command and parses its output for enabled/loaded plugins.
+
+    :return: A list of enabled plugins.
+    :rtype: List
+    """
+
+    # We rely on yum itself to report what plugins are used when it is invoked.
+    # An alternative approach would be to check /usr/lib/yum-plugins/ (install
+    # path for yum plugins) and parse corresponding configurations from
+    # /etc/yum/pluginconf.d/
+
+    if get_source_major_version() == '7':
+        yum_cmd = ['yum']
+    else:
+        yum_cmd = ['yum', '-v']  # On RHEL8 we need to supply an extra switch
```

Comment:
Since you know it's RHEL8+
```suggestion
        yum_cmd = ['dnf', '-v']  # On RHEL8 we need to supply an extra switch
```

Reply:
Applied in the latest version. Thanks.